docs/lua-api-dimension.md

```diff
@@ -35,27 +102,93 @@ gravity = 0.5
 
 Default value is `1`.
 
-### `id`
+#### `horizontal_sprint_mod`
 
-ID of the dimension. **Mandatory field.**
+Horizontal sprint speed multiplier.
 
 Example:
 ```lua
-id = "dimension_nether"
+horizontal_sprint_mod = 1.5
 ```
 
-IDs are usually of the form `mod:dimension` but the `mod:` prefix is prepended automatically so it's not needed.
+Default value is `1.5`.
 
-### `name`
+#### `is_sneak_always_mod`
 
-Label of the dimension. **Mandatory field.**
+Sprint speed multiplier.
```

Comment:
It's a boolean, not a speed multiplier?

Reply:
I was expecting vertical sprint speed mod to be below the horizontal one, started typing it in, and I guess I forgot to use control Z